BGL/doc/BGL/Concepts/FaceListGraph.h

```diff
@@ -9,33 +9,25 @@ the requirement for traversal of all faces in a graph.
 \cgalHasModel `CGAL::Polyhedron_3`
 \cgalHasModel `CGAL::Surface_mesh`
 
-\cgalHeading{Notations}
-
-<dl>
-<dt>`G`</dt> 	<dd>A type that is a model of `FaceListGraph`.</dd>
-<dt>`g`</dt> 	<dd>An object of type `G`.</dd>
-</dl>
+*/
+class FaceListGraph{};
```

Comment:
For this class, all related functions are using `const FaceListGraph&` while for `FaceGraph` it is only `FaceGraph&`.
Also, the difference with the previous documentation was that the constness was not specified. One solution is to use `FaceGraph&` and say somewhere that `FaceGraph` might be `const FaceGraph_bis `.